.pullapprove.yml

```diff
@@ -8,20 +8,10 @@ group_defaults:
     enabled: false
 groups:
   reviewers:
-    required: 2
+    required: 1
     github_reviews:
       enabled: true
     author_approval:
       ignored: true
-    users:
-        - Horneth
-        - cjllanwarne
-        - geoffjentry
-        - jsotobroad
-        - katevoss
-        - kcibul
-        - kshakir
-        - mcovarr
-        - ruchim
-        - danbills
-        - jbuabbud
+    teams:
+        - redteam
```

Comment:
It's been a redteam convention to always add a newline at the end of files, for two reasons:
- So that when you `cat` them, the cursor ends up on the next line
- (To avoid this ugly mark in github PRs)

I suggest sticking with the convention unless you feel strongly otherwise